app/routes/user/endorsements.tsx

```diff
@@ -466,17 +525,27 @@ export function EndorsementRequestForm() {
             }
           />
         </Grid>
-        <Grid col="auto">
+      </Grid>
+      <Grid row>
+        <div className="tablet:grid-col-fill">
+          <TextInput
+            id="note"
+            name="note"
+            type="text"
+            placeholder="Note for endorser"
+          />
+        </div>
+        <div className="tablet:grid-col-auto">
```

Comment:
There's a way to write this with the `<Grid>` component. Can we do that, please?